Cargo.toml

```diff
@@ -38,6 +38,11 @@ serde_json = "1.0.104"
 [features]
 io-uring = ["actix-web/experimental-io-uring"]
 
+[profile.release]
+codegen-units = 1
+lto = true
+panic = "abort"
```

Comment:
Hey @IdoKendo :wave: 


Thank you for the great work :smile: Can you please share some documentation of these three flags? Or explain what is the purpose of these three flags?

Reply:
codegen-units - https://doc.rust-lang.org/rustc/codegen-options/index.html#codegen-units

Reply:
@IdoKendo , what are the other two here?

Reply:
1. By default, Cargo instructs compilation units to be compiled and optimized in isolation. LTO instructs the linker to optimize at the link stage. This can, for example, remove dead code and often times reduces binary size.
2. By default, when Rust code encounters a situation when it must call panic!(), it unwinds the stack and produces a helpful backtrace. The unwinding code, however, does require extra binary size. rustc can be instructed to abort immediately rather than unwind, which removes the need for this extra unwinding code.
ref: https://doc.rust-lang.org/cargo/reference/profiles.html#default-profiles